qa/packaging/src/test/java/org/elasticsearch/packaging/test/PackagingTestCase.java

```diff
@@ -147,7 +147,7 @@ protected void failed(Throwable e, Description description) {
             failed = true;
             if (installation != null && installation.distribution.isDocker()) {
                 logger.warn("Test {} failed. Printing logs for failed test...", description.getMethodName());
-                FileUtils.logAllLogs(installation.logs, logger);
+                dumpDebug();
```

Comment:
We need to fetch and log all container logs instead. I fixed this [in](https://github.com/elastic/elasticsearch/pull/131294/commits/eaef5dbfd0f2012009d653f74ab4831216828d6a) all [other](https://github.com/elastic/elasticsearch/pull/131295/commits/2efb3085fbc56395bb8579a4acf51baf855a48fb) [branches](https://github.com/elastic/elasticsearch/pull/131293/commits/e13a3740689e0bc46036edfbdb020191816bb616), so including it here as well.